core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -2213,32 +2212,6 @@ private void sendAndReset() throws PeerDisconnectedException, PeerIsSlowToReadEx
         responseAsciiSink.reset();
     }
 
```

Comment:
This method looks like it's meant for copy in - that is copy from client to server . Is it completely wrong ? 
I know we don't rally implement it but maybe we could just remove the call but keep the method for later ?

Reply:
no, the method is not completely wrong. it's for the copy-in sub protocol. we don't support that. 

we support only a local copy, from a protocol perspective then looks like a regular query. 

I can keep it (unused) in the code and add some comments. I had been considering it, but then decided an unused private method was too weird and it's in git anyway. but it's true it's hard to discover it when it's just in git.